Cargo.toml

```diff
@@ -1,21 +1,44 @@
 [workspace]
-members = []
+resolver = "2"
+members = ["engine/bin/*", "engine/lib/*"]
 
-[package]
-name = "ltheory"
-authors = ["Limit Theory Redux Team"]
+[workspace.package]
 version = "0.1.0"
-publish = false
+authors = ["Limit Theory Redux Team"]
 edition = "2021"
-autobins = false
+license = "Unlicense"
+# TODO: do we really need these 2 fields?
+# publish = false
+# autobins = false
+
+[workspace.dependencies]
+# workspace bin crates
+ltr = { path = "engine/bin/ltr" }
 
-[[bin]]
-path = "src/Main.rs"
-name = "ltr"
+# workspace lib crates
+phx = { path = "engine/lib/phx" }
+fmod-sys = { path = "engine/lib/fmod-sys" }
+luajit-ffi-gen = { path = "engine/lib/luajit-ffi-gen" }
 
-[dependencies]
-libc = "0.2"
-phx = { path = "libphx" }
+# release crates
+chrono = { version = "0.4.24", default-features = false }
+flate2 = { version = "1.0.25", default-features = false }
+freetype-sys = { version = "0.17", default-features = false }
+glam = { version = "0.24", default-features = false }
+image = { version = "0.24.6", default-features = false }
```

Comment:
How come we're putting all our dependencies in the root namespace, then just using workspace = true in one place? Pretty much all of these dependencies are only for phx. Can we just keep only common dependencies here?

Reply:
The idea is to declare all dependencies in top level Cargo.toml, doesn't matter if they're used in a single crate or multiple. This way we have a central place for managing dependencies and should not look through subcrates